test/fixtures/access-control/server/server.js

```diff
@@ -5,7 +5,7 @@
 
 var loopback = require('../../../..');
 var boot = require('loopback-boot');
-var app = module.exports = loopback({ localRegistry: true });
+var app = module.exports = loopback({ localRegistry: true, loadBuiltinModels: true });
```

Comment:
> and why are tests failing without this... complaining that the builtin model Application is unknown?

Ah, interesting, my understanding of how this part of LoopBack works was wrong then. I assume the tests are failing because ACLs are resolving `Application` model, but they don't declare this dependency anywhere.

Let's go with what you are proposing here then. I'll delete the other "ditto" comment.